doc/readme.html

```diff
@@ -398,8 +399,21 @@
       options.
     </p>
 
+    <a name="optional-build"/>
+      <h4>Selecting optional compilation features</h4>
```

Comment:
in the ToC, you called this "Selecting optional *build* features". Want to settle on one or the other?